doc/source/support/frequently-asked-questions.rst

```diff
@@ -247,6 +247,7 @@ Instead of manually running the ``sphinx-apidoc -o ..`` command from your termin
             "sphinx.ext.viewcode",
             "sphinx.ext.intersphinx",
             "sphinx_rtd_theme",
+            "sphinx_copybutton",
```

Comment:
This is the extension for copying code block that is also configured in the cookiecutter repo